news/2025/2025-04-15-the-followpoint-azer-the-statistics-breaking-magician.md

```diff
@@ -0,0 +1,305 @@
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+In your opinion, what's the current state of the tournament scene? Anything that you think can be improved, or everything seems fine?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/2155578" style="background-image: url('/wiki/shared/avatars/Azer.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/2155578">Azer</a></p>
+
+I think we are due for some more high profile events again! It feels like there aren't really that many, but I think an important thing to figure out right now is how to make open rank tournaments exciting again. I think people tend to avoid them due to superteams spoiling the experience a bit, so I'm excited to see if a solution to that gets implemented anywhere so we don't have to wait months between big events. Non S-tier events seem to be doing fine, but there isn't much outside of OWC to get really excited over. I think the scene might be due for another big shake up, which is why we're trying a lot of experimental things with [Lazer Grand Arena](/wiki/Tournaments/LGA/2024), perhaps something there ends up sticking. There's a lot of very high quality events with very little excitement, relative to the quality they present. I don't really know the solution to any of that, but I think it's definitely the biggest ongoing issue with the tournament scene.
+
+---
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+Many know you as a World Cup organiser, but you've actually participated and won a bunch of tournaments too. So what do you feel is your best moment coming from a tournament player perspective?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/2155578" style="background-image: url('/wiki/shared/avatars/Azer.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/2155578">Azer</a></p>
+
+It's pretty hard to pick one after playing for this long, but I can cut it down to 3 that mean a lot to me!
+
+[OWC 2015 vs Taiwan: United Tiebreaker](https://www.youtube.com/watch?v=YGOxH7j8YwA)
```

Comment:
these videos should be embedded imo

Reply:
Alright, will do

Reply:
Update: 3rd video link doesn't want to work as an embed for some reason, so I'm not sure what to do about this